malariagen_data/anoph/snp_data.py

```diff
@@ -1611,7 +1613,7 @@ def biallelic_snp_calls(
 
         with self._spinner("Prepare biallelic SNP calls"):
             # Subset to biallelic sites.
-            ds_bi = ds.isel(variants=loc_bi)
+            ds_bi = dask_compress_dataset(ds, indexer=loc_bi, dim="variants")
```

Comment:
This is the main optimisation. Our implementation of `dask_compress_dataset()` is much faster than calling `ds.isel()`.